### PR TITLE
Fixed some French translations

### PR DESCRIPTION
**Pull request by NathanBnm**

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -49,11 +49,11 @@ msgstr "Chercher dans le carnet courant"
 
 #: /home/dabou/Projets/github/Notes-up/po/../src/Widgets/NewNotebookDialog.vala:55
 msgid "Name:"
-msgstr "Nom:"
+msgstr "Nom :"
 
 #: /home/dabou/Projets/github/Notes-up/po/../src/Widgets/NewNotebookDialog.vala:56
 msgid "Color:"
-msgstr "Couleur:"
+msgstr "Couleur :"
```
[the remark this answers is not in the document]
Comment (NathanBnm) [reply po/fr.po]: In French the colon must be preceded by a space :+1:
For further information you can read this article. French punctuation has special rules : https://www.iwillteachyoualanguage.com/learn/french/french-tips/french-punctuation

File: po/fr.po [hunk 2]
```diff
@@ -335,7 +335,7 @@ msgstr "<Ctrl>Q"
 
 #: /home/felipe/Code/Notes-up/po/../src/Application.vala:39
 msgid "<Ctrl><Shift>N"
-msgstr "<Ctrl><Shift>N"
+msgstr "<Ctrl><Maj>N"
```
[the remark this answers is not in the document]
Comment (NathanBnm) [reply po/fr.po]: I don't really understand :confused: In French the "Shift" key doesn't exists. It's called the "Maj" key on a keyboard.

**Comment by NathanBnm** [comment]
I just resolved them :smiley: I was right, I left you explanations.

File: po/fr.po (hunk 2 above)
[the remark this answers is not in the document]
Comment (Philip-Scott) [reply po/fr.po]: The translation might be correct, but this is for internal purposes to get the shortcut working in code. If you change it to Maj it won't work

[the remark this answers is not in the document]
Comment (NathanBnm) [reply po/fr.po]: Okay understand! So why is this string in translation files?

**Comment by NathanBnm** [comment]
@Philip-Scott I've made the changes, and just fixed a few things

**Pull request merged**